tests/test.cpp

```diff
@@ -863,7 +864,7 @@ void NativeTypeTest() {
 // Guard against -Wunused-function on platforms without file tests.
 #ifndef FLATBUFFERS_NO_FILE_TESTS
 // VS10 does not support typed enums, exclude from tests
-#if !defined(_MSC_VER) || _MSC_VER >= 1700
+#  if !defined(_MSC_VER) || _MSC_VER >= 1700
```

Comment:
if you are running this on a google machine, it uses a different clang format with a bug in it. The usual workaround is just to run clang-format twice.

Reply:
I've tried ran it again, but doesn't seem to help, I'll just manually revert these.